specs/namespace/creation-for-whitelist.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
There is also a few other cases that you can find in `specs/class` that needs to be accounted for. For example it should also apply to:

- abstract classes
- interfaces
- ~~traits~~: they cannot be whitelisted; may require a test that it doesn't work there still
- ~~anonymous~~: they cannot be whitelisted; may require a test that it doesn't work there still